K2Bridge/Models/Request/Aggregations/Bucket/Histogram/Bounds.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Models.Request.Aggregations
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Extended_bounds can "force" the histogram aggregation to start building buckets on a specific min value
```

Comment:
All the docs here are about extended bounds, even though we don't support them at all. We do support hard_bounds, so add docs for that.